### PR TITLE
Migrate pkg/controller/volume/persistentvolume/testing/testing.go to structured logging

### PR DESCRIPTION
**Pull request by CaoDonghui123**
#### What type of PR is this?

/kind feature
#### What this PR does / why we need it:
Structured Logging migration:
pkg/controller/volume/persistentvolume/testing/testing.go
xref:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
#### Which issue(s) this PR fixes:

part of kubernetes/enhancements#1602

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/controller/volume/persistentvolume/testing/testing.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 		}
 		r.changedObjects = append(r.changedObjects, volume)
 		r.changedSinceLastSync++
-		klog.V(4).Infof("created volume %s", volume.Name)
+		klog.V(4).InfoS("Created volume", "name", volume.Name)
```
Comment (shubheksha) [inline pkg/controller/volume/persistentvolume/testing/testing.go:127]:
```suggestion
		klog.V(4).InfoS("Created volume", "volumeName", volume.Name)
```

File: pkg/controller/volume/persistentvolume/testing/testing.go [hunk 2]
```diff
@@ -144,7 +144,7 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 		}
 		r.changedObjects = append(r.changedObjects, claim)
 		r.changedSinceLastSync++
-		klog.V(4).Infof("created claim %s", claim.Name)
+		klog.V(4).InfoS("Created claim", "name", claim.Name)
```
Comment (shubheksha) [inline pkg/controller/volume/persistentvolume/testing/testing.go:147]:
```suggestion
		klog.V(4).InfoS("Created claim", "claimName", claim.Name)
```

File: pkg/controller/volume/persistentvolume/testing/testing.go [hunk 3]
```diff
@@ -160,7 +160,7 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 				return true, obj, ErrVersionConflict
 			}
 			if reflect.DeepEqual(storedVolume, volume) {
-				klog.V(4).Infof("nothing updated volume %s", volume.Name)
+				klog.V(4).InfoS("Nothing updated volume", "name", volume.Name)
```
Comment (shubheksha) [inline pkg/controller/volume/persistentvolume/testing/testing.go:163]:
```suggestion
				klog.V(4).InfoS("Nothing updated volume", "volumeName", volume.Name)
```

File: pkg/controller/volume/persistentvolume/testing/testing.go [hunk 4]
```diff
@@ -177,7 +177,7 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 		r.volumes[volume.Name] = volume
 		r.changedObjects = append(r.changedObjects, volume)
 		r.changedSinceLastSync++
-		klog.V(4).Infof("saved updated volume %s", volume.Name)
+		klog.V(4).InfoS("Saved updated volume", "name", volume.Name)
```
Comment (shubheksha) [inline pkg/controller/volume/persistentvolume/testing/testing.go:180]: Same as above

**Review by shubheksha — CHANGES_REQUESTED** [review]
Could you replace "name" with `volumeName` and `claimName` across the PR for greater clarity please? Thank you!

**Comment by yangjunmyfm192085** [comment]
duplicated with 
https://github.com/kubernetes/kubernetes/pull/100831
/wg structured-logging

**Pull request closed without merging**